### PR TITLE
doc: tutorial fixes

### PR DESCRIPTION
**Pull request by fscherf**
This PR fixes some typos and wording problems in the tutorial. It also adds some nicer numbering

### CODE REVIEW

File: doc/content/tutorial/01-getting-started/index.rst [hunk 1]
```diff
@@ -3,18 +3,42 @@
...
+is that Lona projects contain multiple files, and scripts can define a whole
+Lona app, using only one file. Both structures have the same features, and can
+easily be migrated from one to another.
 
 In this tutorial we will focus on scripts, because they are easier to setup,
 and the examples are easier to copy-paste.
 
-Lona is a web application server and rendering framework, so it comes with a
-very bland (or lack of) styling. We will use
+The Lona comes with a very bland (or lack of) styling. We will use
```
Comment (hut) [inline doc/content/tutorial/01-getting-started/index.rst:39]: The Lona?

File: doc/content/tutorial/01-getting-started/index.rst [hunk 2]
```diff
@@ -3,18 +3,42 @@
...
+**More information:**
+ - `Frontend and Backend (Wikipedia) <https://en.wikipedia.org/wiki/Frontend_and_backend#Back-end_focused>`_
+
+
+Apps, Projects and Scripts
+--------------------------
+
+In this tutorial we will create and discuss multiple Lona "apps" (short for
+"application"). A Lona app consists of one or more "views", which are Python
+classes, that render one page of Lona created a website. Every example in this
```
Comment (hut) [inline doc/content/tutorial/01-getting-started/index.rst:26]: that's not really a sentence :)

File: doc/content/tutorial/01-getting-started/index.rst [hunk 3]
```diff
@@ -3,18 +3,42 @@
...
+Introduction
+------------
+
+Lona is a full-stack web-application framework. That means it is possible to
+write a whole, self-contained, web application in pure Python. Lona runs
+completely on the server. Traditionally that would mean that Lona could only do
+"backend" tasks, like talking to a database, but no "frontend" tasks like
+responding to a click event in the browser. Lona implements a generic frontend
+that can be controlled by the backend, live,
+over websockets.
```
Comment (hut) [inline doc/content/tutorial/01-getting-started/index.rst:15]: Great intro :) You might want to join line 14 and 15

File: doc/content/tutorial/01-getting-started/index.rst (hunk 1 above)
Comment (fscherf) [reply doc/content/tutorial/01-getting-started/index.rst]: The Lona! Fixed :)

File: doc/content/tutorial/01-getting-started/index.rst (hunk 2 above)
Comment (fscherf) [reply doc/content/tutorial/01-getting-started/index.rst]: Whoops. Now it is

File: doc/content/tutorial/01-getting-started/index.rst (hunk 3 above)
Comment (fscherf) [reply doc/content/tutorial/01-getting-started/index.rst]: Thanks! :)
Yes, you are right. I joint both lines

**Review by hut — APPROVED** [review]

**Comment by fscherf** [comment]
@hut: Thanks for reviewing! :)

**Pull request merged**